rules/REQUEST-933-APPLICATION-ATTACK-PHP.conf

```diff
@@ -128,10 +128,14 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
     severity:'CRITICAL',\
     setvar:'tx.933120_tx_0=%{tx.0}',\
     chain"
-    SecRule MATCHED_VARS "@pm =" \
+    SecRule MATCHED_VARS "@rx \b([^\s]+)\s*=" \
         "capture,\
-        setvar:'tx.php_injection_score=+%{tx.critical_anomaly_score}',\
-        setvar:'tx.inbound_anomaly_score_pl1=+%{tx.critical_anomaly_score}'"
+        setvar:'tx.933120_possible_directive=%{TX.1}',\
+        chain"
+        SecRule TX:933120_possible_directive "@pmFromFile php-config-directives.data" \
+            "capture,\
```

Comment:
I do not think we need this capture.

Reply:
> I do not think we need this capture.

Probably not but in that case we should use `%{TX.1}` in logdata action.

Reply:
@ssigwart Can you look also at this again? Try removing `capture` and using TX.1 in `logdata`, should work.

Reply:
I updated it. I'm not all that familiar with `TX` and `capture`, so please let me know if it needs more updates.